msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -402,9 +327,9 @@ The following code shows an example of sending a simple Adaptive Card:
 }
 ```
 
-### Form completion feedback
+#### Form completion feedback
 
-Form completion message appears in Adaptive Cards while sending a response to the bot. The message can be of two types, error or success:
+You can build form completion feedback using adaptive card. Form completion message appears in Adaptive Cards while sending a response to the bot. The message can be of two types, error or success:
```

Comment:
This sentance reads slightly strangely. Perhaps "You can build form completion feedback using an adaptive card." is more gramattically correct

Reply:
Updated as mentioned.